interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -920,4 +931,38 @@ protected static void assertSuccess(StreamRecorder<?> recorder) {
       throw new AssertionError(recorder.getError());
     }
   }
+
+  /** Helper for asserting remote address {@link io.grpc.ServerCall#attributes()} */
+  protected void assertRemoteAddr(String expectedRemoteAddress) {
+    TestServiceGrpc.TestServiceBlockingStub stub = TestServiceGrpc.newBlockingStub(channel)
+            .withDeadlineAfter(5, TimeUnit.SECONDS);
+
+    stub.unaryCall(SimpleRequest.getDefaultInstance());
+
+    HostAndPort remoteAddress = HostAndPort.fromString(serverCallCapture.get().attributes()
+            .get(GrpcUtil.REMOTE_ADDR_CALL_ATTR_KEY).toString());
+    assertEquals(expectedRemoteAddress, remoteAddress.getHostText());
+  }
+
+  /** Helper for asserting TLS info in SSLSession {@link io.grpc.ServerCall#attributes()} */
+  protected void assertTlsInfo(String tlsInfo) {
```

Comment:
Maybe name it `assertX500SubjectDn` or `assertTlsSubjectDn`? "TlsInfo" is pretty vague, and I wouldn't have expected it to mean the Subject DN.